pkg/cli/config/config_test.go

```diff
@@ -1,35 +1,23 @@
 package config
 
 import (
+	"fmt"
 	"testing"
-
-	"github.com/openshift/backplane-cli/pkg/info"
 )
 
-func TestGetBackplaneConfigFile(t *testing.T) {
-	t.Run("it returns the Backplane configuration file path if it exists in the user's env", func(t *testing.T) {
-		t.Setenv(info.BACKPLANE_CONFIG_PATH_ENV_NAME, "~/.backplane.stg.env.json")
-		path, err := GetBackplaneConfigFile()
+func TestGetBackplaneConfig(t *testing.T) {
+	t.Run("it returns the user defined proxy instead of the configuration variable", func(t *testing.T) {
+		userDefinedProxy := "example-proxy"
+		t.Setenv("HTTPS_PROXY", userDefinedProxy)
+		config, err := LoadConfig()
 		if err != nil {
 			t.Error(err)
 		}
-		if path != "~/.backplane.stg.env.json" {
-			t.Errorf("expected path to be %v, got %v", "~/.backplane.stg.env.json", path)
-		}
-	})
 
-	t.Run("it returns the default configuration file path if it does not exist in the user's env", func(t *testing.T) {
-		path, err := GetBackplaneConfigFile()
-		if err != nil {
-			t.Error(err)
-		}
+		fmt.Println(config.ProxyURL)
```

Comment:
hope this is a test print

Reply:
Ah yes, the real way to test things :wink: